src/prted/pmix/pmix_server.c

```diff
@@ -1180,8 +1180,8 @@ static void pmix_server_dmdx_recv(int status, pmix_proc_t *sender, pmix_data_buf
             pmix_asprintf(&req->operation, "DMDX: %s:%d", __FILE__, __LINE__);
             req->proxy = *sender;
             memcpy(&req->tproc, &pproc, sizeof(pmix_proc_t));
-            req->info = info;
-            req->ninfo = ninfo;
+            req->info = NULL;
+            req->ninfo = 0;
```

Comment:
Wait - that's wrong. We _do_ need the info later when we are evicted from the hotel and have to try and get the info again.

Reply:
Then I will remove the fre on line 1191 instead. This is pointing to a free'd pointer otherwise.